dashboard/app/api.go

```diff
@@ -104,7 +104,12 @@ func handleAPI(c context.Context, r *http.Request) (reply interface{}, err error
 	client := r.PostFormValue("client")
 	method := r.PostFormValue("method")
 	log.Infof(c, "api %q from %q", method, client)
-	subj := determineAuthSubj(c, r.Header["Authorization"])
+	auth := mkAuthEndpoint(googleTokenInfoEndpoint)
+	subj, err := auth.determineAuthSubj(r.Header["Authorization"])
+	if err != nil {
+		log.Errorf(c, "%v", err)
```

Comment:
The returned error should be logged by handleJSON, so this function don't need to.